css/layout.css

```
@@ -0,0 +1,70 @@
+body {
+    display: grid;
+    grid-template-columns: 1fr;
+}
+
+.container {
+    display: grid;
+    grid-template-columns: 1fr 200px 600px 200px 1fr;
+    /* grid'as savyje gali turėti kitų gridų'ų (grid'o cell'ė) */
+    /* grid-template-columns: 1fr 1fr 1fr; - taip visi stulpeliai tampa vienodo dydžio */
+    padding: 40px 0;
+    border: green 3px solid;
```

pasiūlymas, kaip pakeisti tą vietą
```suggestion
    border: red 7px solid;
```